server/candlepin.spec.tmpl

```diff
@@ -127,6 +126,7 @@ sed -i 's/#LOGROTATE-3.8#//' %{buildroot}%{_sysconfdir}/logrotate.d/%{name}
 
 %{__install} -m 755 code/setup/cpsetup %{buildroot}/%{_datadir}/%{name}/cpsetup
 %{__install} -m 755 code/setup/cpdb %{buildroot}/%{_datadir}/%{name}/cpdb
+%{__install} -m 755 bin/liquibase.sh %{buildroot}/%{_datadir}/%{name}/liquibase.sh
```

Comment:
This needs a corresponding entry in the `%files` section.

Please add `%{_datadir}/%{name}/liquibase.sh` to the `%files` section.

Reply:
Done.